### PR TITLE
Adding tests for the user cluster prometheus metrics

### PR DESCRIPTION
**Pull request by moadqassem**
**What this PR does / why we need it**:
Adding e2e tests to ensure prometheus metrics are available.

**Which issue(s) this PR fixes**
Fixes #kubermatic/kubermatic#3424

**Does this PR introduce a user-facing change?**:
No

```release-note
None
```

### CODE REVIEW

File: api/cmd/conformance-tests/custom_tests.go [hunk 1]
```diff
@@ -246,3 +255,119 @@ func (r *testRunner) testLB(log *zap.SugaredLogger, userClusterClient ctrlruntim
...
+		Name:      "prometheus-0",
+	}
+
+	prometheusPod := &corev1.Pod{}
+	if err := seedClient.Get(context.Background(), namespacedPod, prometheusPod); err != nil {
+		return fmt.Errorf("failed to get prometheus pod: %v", err)
+	}
+
+	path := fmt.Sprintf("/api/v1/namespaces/%s/pods/%s/portforward", cluster.Status.NamespaceName, prometheusPod.Name)
+	hostIP := strings.TrimLeft(r.seedRestConfig.Host, "htps:/")
```
Comment (youssefazrak) [inline api/cmd/conformance-tests/custom_tests.go:278]: Typo, shoud be _https_ not _htps_

Comment (moadqassem) [reply api/cmd/conformance-tests/custom_tests.go:278]: https://golang.org/pkg/strings/#TrimLeft ;)

**Pull request merged**